components/Socket.tsx

```diff
@@ -28,27 +28,23 @@ export const Opcodes = {
 
 const DEFAULT_WS_SERVER = "ws://localhost:8080";
 const DEFAULT_WS_CLOSURE = 1000;
+const DEFAULT_PING_INTERVAL = 500;
```

Comment:
I only have a minor nitpick here where I think all constants/variables should have their units (i.e. `DEFAULT_PING_INTERVAL_MSEC`) to make it better for readability